infra-parent/infra-v2/src/main/java/com/redhat/service/smartevents/infra/v2/api/models/dto/BridgeStatusDTO.java

```diff
@@ -16,12 +16,12 @@ public class BridgeStatusDTO {
     private long generation;
 
     @JsonProperty("conditions")
-    private Set<ConditionDTO> conditions;
+    private List<ConditionDTO> conditions;
```

Comment:
Why this change?
Do we expect duplicates conditions?

Reply:
no, we don't have dublicate in conditions. I change it from `Set` to `List` to keep it consistent with `ProcessorStatusDTO` implementation.